crates/proto/src/rr/dnssec/nsec3.rs

```diff
@@ -103,9 +103,10 @@ use crate::serialize::binary::{BinEncodable, BinEncoder};
 ///    requires IETF Standards Action [RFC2434].
 /// ```
 #[cfg_attr(feature = "serde", derive(Deserialize, Serialize))]
-#[derive(Debug, PartialEq, Eq, Hash, Clone, Copy)]
+#[derive(Debug, PartialEq, Eq, Hash, Clone, Copy, Default)]
 pub enum Nsec3HashAlgorithm {
     /// Hash for the Nsec3 records
+    #[default]
     SHA1,
```

Comment:
is this the only hash supported still?

Reply:
yes, I doubt it will change but most tools related to DNSSEC still insist in keeping an algorithm flag that's set to SHA-1 by default :shrug: